.github/workflows/release.yml

```diff
@@ -96,7 +96,46 @@ jobs:
         if: contains(matrix.arch, 'linux') && endsWith(matrix.arch, '-gnu')
         run: |
           sudo apt-get -y update
-          sudo apt-get -y install libssl-dev pkg-config g++-aarch64-linux-gnu gcc-aarch64-linux-gnu
+          sudo apt-get -y install libssl-dev pkg-config g++-aarch64-linux-gnu gcc-aarch64-linux-gnu binutils-aarch64-linux-gnu
+
+      - name: Compile Python 3.10.10 from source for Aarch64
+        if: contains(matrix.arch, 'aarch64-unknown-linux-gnu')
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install wget
+          wget https://www.python.org/ftp/python/3.10.10/Python-3.10.10.tgz
```

Comment:
suggestion: Can you make the process of compiling python-3.10 the standalone script? For example, you can put those instructions in `./docker/compile-python.sh`. Then we also can use the script in Dockerfile.